### DISCUSSION TITLE
Html entities combining entitie

### DISCUSSION THREAD

**Discussion by SwykanySolutions** [dompdf/dompdf#3158] [Q&A]
Hello Everyone, i need assistance.
I need write text with combine html entities, but, the text does not appear correctly, the place of the element appears a question mark.
And is it possible to add elements combining, the available fonts like helvelica among others?
Next I will put the text that appears after the generation of the pdf: P?G?L?S?x

**Comment by bsweeney** [comment]
You need to make sure you're using a font that supports the characters you want to display. The core PDF fonts (Helvetica, Times Roman, Courier) and the generic font families (sans-serif, serif, monospace) only support windows-ansi encoded characters.

See the wiki for more information:
- https://github.com/dompdf/dompdf/wiki/About-Fonts-and-Character-Encoding
- https://github.com/dompdf/dompdf/wiki/UnicodeHowTo

Try one of the bundled DejaVu fonts:
```
<div style="font-family: DejaVu Sans;">Ђ</div>
```

**Reply by SwykanySolutions** [reply]
Thanks, but i need help with another error.
This error is:
![image](https://user-images.githubusercontent.com/126429370/234621217-7b7b747a-6246-4c3a-9b3d-a648be5068c4.png)
This image contains the error. I need help for solution this problem.
This problem ocurred, when i using a html entitie (i&768;) and the another entitie is (&icute;)

**Reply by bsweeney** [reply]
I'm not sure what's going on with the grave accent. I'd have to look into it but it looks like the glyf instructions adjust it's position so that it renders over the previous character.

That being said, you don't want to use a standard "i" if you also want to display that accent. For the i with grave accent use `&igrave;`. And just to confirm the i with acute accent should be `&iacute;`.
